### ISSUE TITLE
Prettifying Large Numbers from `v1` JSONs in `v2` Auspice

### ISSUE THREAD

**Issue by emmahodcroft** [nextstrain/auspice#809]
If continues values are too large, this breaks the 'prettify' function used by Auspice `v2` to render `v1` JSONs.

This is in `convertJsonSchemas.js` lines 35-36.
https://github.com/nextstrain/auspice/blob/v2/cli/server/convertJsonSchemas.js#L35

A large number like 185,781 ends up sending a negative value to `toFixed`, which breaks Auspice. Confusingly, it doesn't give a very helpful error, simply saying the JSONs could not be loaded. 

Tied in with this issue is potentially also an issue in displaying discrete values which happen to be numbers (cannot be set to `ordinal` in `v1` JSONs), whereby the number of zeros displayed is very large:
![image](https://user-images.githubusercontent.com/14290674/66737202-9fb71d00-ee6b-11e9-806b-c43d66a339e1.png)
For an actual value of 0, you get 15 trailing zeros which looks silly:
![image](https://user-images.githubusercontent.com/14290674/66737230-acd40c00-ee6b-11e9-9d35-20265a8ee27c.png)

I ended up resolving both issues with the following code:
```
const decVal = Math.abs(val) - Math.abs(Math.floor(val));
const magnitude = decVal !== 0 ? Math.abs(Math.ceil(Math.log10(Math.abs(decVal)) + 1e-10)) : 0;
return multiplier ? val.toFixed(magnitude) + "\u00D7" : val.toFixed(magnitude);
```

**HOWEVER** I believe this may not work properly for values <1. 

For those on the Slack, the full discussion is here:
https://bedfordlab.slack.com/archives/C116R9PR7/p1569944031018300

**Comment by jameshadfield** [comment]
Thanks @emmahodcroft  -- this code (in the server of auspice v2) is the same as the code in the client of v1: https://github.com/nextstrain/auspice/blob/5a6d15bdf8c2148c500ee343418ffd3fd5c985b9/src/util/stringHelpers.js#L34, although they may be employed in different situations. Does this same behavior occur using v1 JSONs in auspice v1?

**Comment by jameshadfield** [comment]
closed by f116253
